src/Sylius/Bundle/WebBundle/Resources/translations/messages.en.yml

```diff
@@ -555,6 +555,7 @@ sylius:
         provinces: Provinces
         qty_max: 'Qty max'
         qty_min: 'Qty min'
+        quantity: 'Qty'
```

Comment:
New translations go to SyliusUiBundle, changes to existing in WebBundle (for now).